packaging/npm/src/cli.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
```

Comment:
the compiled js will be sym-linked by npm (because we defined a "bin" value in the package.json) to the user's path so they should be able to execute defang and it'll execute the cli.js file. Kind of an unecessary step as we are just forwarding it to the binary anyways but we don't have the binary until the postInstall step completes.